src/engine/renderer/tr_shader.cpp

```diff
@@ -3571,7 +3578,51 @@ static bool ParseShader( const char *_text )
 
 			continue;
 		}
-		// parallax mapping
+		// normalFormat <format>
+		//
+		// expects OpenGL format as default
+		//
+		// OpenGL format is described there:
+		//     https://github.com/KhronosGroup/glTF/tree/2.0/specification/2.0#materialnormaltexture
+		//
+		// > The normal vectors use OpenGL conventions where +X is right and +Y is up.
+		// > +Z points toward the viewer.
+		//
+		// example of formats:
+		//
+		//  1  1  1  OpenGL format (default)
+		//  1 -1  1  DirectX format with reverted green (Y) channel
+		// -1  1  1  weird other format with reverted red (X) channel
+		else if ( !Q_stricmp( token, "normalFormat" ) )
+		{
+			for ( int i = 0; i < 3; i++ )
+			{
+				token = COM_ParseExt2( text, false );
+
+				if ( token[ 0 ] == 0 )
```

Comment:
Since `token[0]` is a char it would be nicer to do `!token[0]` or `token[0] == '\0'` instead of comparing to an int.

Reply:
you're right, and I'm especially the kind of guy that likes explicit correct comparisons, I'll do that :-)

Reply:
In fact I copy pasted this line, there is 51 of them in that file. I fixed it (will push) but I may fix the 51 others in a bikeshedding commit in the future.